package.json

```diff
@@ -27,7 +27,7 @@
     "@rjsf/validator-ajv8": "5.5.0",
     "autosuggest-highlight": "3.3.4",
     "buffer": "6.0.3",
-    "check-dependencies": "1.1.0",
+    "check-dependencies": "^1.1.0",
```

Comment:
Also, please remove that change from the PR. node makes that change when you install dependencies, but it's not actually needed.